modules/exploits/multi/postgres/postgres_createlang.rb

```diff
@@ -168,8 +175,11 @@ def load_procedural_language(language, extension)
     match_exists = load_language[:sql_error].match(/(?:(extension|language) "pl#{language}u" already exists)/m)
     return :exists if match_exists
 
-    match_error = load_language[:sql_error].match(/(?:could not (?:open extension control|access) file|unsupported language)/m)
+    match_error = load_language[:sql_error].match(/(?:[Cc]ould not (?:open extension control|access) file|unsupported language)/m)
     return :not_exists if match_error
+
+    # Default to something sane
+    :not_exists
```

Comment:
```suggestion
    :not_exists
```
I think this can be simplified?
at the minute you're returning `:not_exists` if `match_error` is truthy...and then if it's not truthy you end up returning `:not_exists` anyway
it actually kinda makes the whole `match_error` bit pointless since it doesn't matter whether it finds the error or not, do you know what makes use of this return value? that might give us a clue as to what was intended here

Reply:
I'm willing to change it to something else, maybe `:failed` or whatever a good alternative would be 👍 
The return value is being used in:
```ruby
def load_extension?(language)
    case load_procedural_language(language, 'LANGUAGE')
    when ...
    when ...
end
```